src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -994,7 +997,7 @@ function GlobeControls(view, target, domElement, viewerDiv, radius, getPickingPo
         if (this.enableDamping) {
             if (state === CONTROL_STATE.ORBIT && (sphericalDelta.theta > EPS || sphericalDelta.phi > EPS)) {
                 player.play(dampingOrbitalMvt).then(() => this.resetControls());
-            } else if (state === CONTROL_STATE.MOVE_GLOBE && lastRotation.length === 2 && !(lastRotation[1].equals(lastRotation[0]))) {
+            } else if (state === CONTROL_STATE.MOVE_GLOBE && lastRotation.length === 2 && (Date.now() - lastTimeMouseMove < 50) && !(lastRotation[1].equals(lastRotation[0]))) {
```

Comment:
It stayed in the documentation style of this class ;)  (which is a temporary class from what @gchoqueux said).
Though I agree it's never bad to put too much comments, I'll rebase